Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/CarbonBlackEndpointStandard.py

```diff
@@ -951,41 +933,47 @@ def test_module(client: Client, params: dict) -> str:
     :return: 'ok' if test passed, anything else will fail the test.
     :rtype: ``str``
     """
-    try:
 
-        is_fetch = params.get('isFetch')
-
-        """There is 2 sets of api_key&api_secret_key 1 for all API's and 1 for the policy API.
-        check which set of keys to test (organization_key is not needed for the policy pair of keys).
-        at least one of the 2 sets is required.
-        Fetch uses the general api_key."""
-        if (client.api_key and client.api_secret_key and client.organization_key) or \
-                (client.policy_api_key and client.policy_api_secret_key and not is_fetch):
-            if client.api_key or client.api_secret_key or client.organization_key:
-                client.test_module_request()
-                if is_fetch:
-                    filters = fetch_incident_filters(params)
-                    client.search_alerts_request(suffix_url_path=filters.get('suffix_url_path'),
-                                                 minimum_severity=filters.get('min_severity'),
-                                                 policy_id=filters.get('policy_id'),
-                                                 device_username=filters.get('device_username'),
-                                                 device_id=filters.get('device_id'),
-                                                 query=filters.get('query'),
-                                                 alert_category=filters.get('category'))
-            if client.policy_api_key or client.policy_api_secret_key:
-                client.policy_test_module_request()
-            message = 'ok'
-        else:
+    is_fetch = params.get('isFetch')
+
+    # if is_fetch = true and custom API key's is no provided
+    if is_fetch and any(a == "" for a in [client.api_key, client.api_secret_key, client.organization_key]):
```

Comment:
```suggestion
    if is_fetch and not(client.api_key and client.api_secret_key and client.organization_key):
```
We need to also cover the case in which one of them is none, this should work